library/src/CMakeLists.txt

```diff
@@ -80,6 +86,12 @@ add_library(hiptensor SHARED
 add_library(hiptensor::hiptensor ALIAS hiptensor)
 
 target_compile_options(hiptensor PRIVATE ${CMAKE_CXX_FLAGS})
+if(HIPTENSOR_BUILD_COMPRESSED_DBG)
```

Comment:
Do we need to carry this through the interface? Not sure if we want users to require this as well?

Reply:
tested and PRIVATE worked.

Large obj file are from these contraction instance cpp. So -gz should not be inherited.